static/app/components/replays/breadcrumbs/utils.tsx

```diff
@@ -3,12 +3,26 @@ import {BreadcrumbType, Crumb} from 'sentry/types/breadcrumbs';
 /**
  * Generate breadcrumb descriptions based on type
  */
-export function getDescription(crumb: Crumb) {
+export function getDescription(crumb: Crumb, startTimestampMs?: number) {
+  if (crumb.data && 'action' in crumb.data) {
+    switch (crumb.data.action) {
+      case 'largest-contentful-paint':
+        if (crumb.timestamp && startTimestampMs) {
+          return `${new Date(crumb.timestamp).getTime() - startTimestampMs}ms`;
+        }
+        break;
+      default:
+        break;
+    }
+  }
+
   switch (crumb.type) {
     case BreadcrumbType.NAVIGATION:
       return `${crumb.data?.to ?? ''}`;
     case BreadcrumbType.DEFAULT:
       return JSON.stringify(crumb.data);
+    case BreadcrumbType.DEBUG:
+      return crumb.message || '';
```

Comment:
No longer necessary

Reply:
Not following, is LCP the only DEBUG breadcrumb type?

Reply:
we don't need this case since it will fall to the default case